test/unit/auth.js

```diff
@@ -39,15 +40,20 @@ describe('auth', () => {
     });
 
     it('updates user profile for existing user', done => {
-      // NOTE: How do we know that this is a  previous user that has been updated?
-      const userData = users[0];
-      userData.epost = 'ukjent@ukjentsen.com';
-      auth.setOrUpdateUserData(0, userData).then(user => {
-        assert.equal(user._id, userData.sherpa_id);
-        assert.equal(user.navn, `${userData.fornavn} ${userData.etternavn}`);
-        assert.equal(user.epost, userData.epost);
-        done();
-      });
+      // Set userData to the fixture user matching test case user id
+      const userData = users[1];
+      User.findOne({ _id: 1234 })
```

Comment:
Du trenger strengt tatt ikke å hente denne fra databasen først. Hvis du importerer `../fixtures/users` så skal alle de brukeren ligge i databasen før hver test. Jeg ville heller kjørt `findOne` etter `setOrUpdateUserData` for å sjekke at data faktisk ble lagret i databasen.